common.mk

```diff
@@ -74,6 +74,10 @@ check_aws: check_python
 		false; \
 	fi
 
+.PHONY: rename_resources
+rename_resources:
+	python $(project_root)/scripts/rename_resources.py
```

Comment:
I thought I already left a comment about this in my review, but I can't find it. Essentially: this target should depend on the `init` target in `terraform/Makefile`. That probably means this target needs to be moved to that Makefile.

Reply:
This shouldn't be in common.mk. That file is for targets that should be available in every other Makefile.

And I think @jessebrennan is right about the dependency on the init target in `terraform/Makefile`. You should try running this script after running `make clean` in that directory. You can add a convenience target in the root Makefile that calls this target in the `terraform/Makefile`. There is already precedent for that.

Reply:
Now I get it. This IS mentioned again in the two terraform Makefiles. Still, I don't this should be here because it makes it available to every other Makefile and shows up in tab completion. Don't worry about Gitlab for now since you can't test it. Just define this target in the terraform/Makefile.

Reply:
Now I get it. This IS mentioned again in the two terraform Makefiles. Still, I don't this should be here because it makes it available to every other Makefile and shows up in tab completion. Don't worry about Gitlab for now since you can't test it. Just define this target in the terraform/Makefile.